BAZProject/Modules/MovieDetailModule/MovieDetailInteractor.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MovieDetailInterceptor.swift
+//  BAZProject
+//
+//  Created by Leobardo Gama Muñoz on 02/02/23.
+//
+
+import Foundation
+
+final class MovieDetailInteractor {
+    weak var presenter: MovieDetailInteractorOutputProtocol?
+    
+    var movieApiData: DataHelper = DataHelper()
+    var data: Result?
+    let saveData: SaveMovies = SaveMovies()
+    
+}
+
+extension MovieDetailInteractor: MovieDetailInterceptorInputProtocol {
+    func saveMovie() {
+        debugPrint("test of saveMovie")
```

Comment:
Maybe this was useful for testing, but it does not help on productive code, remove the prints you are no longer using or add a `TODO: - Remove print once this is working properly` if you are still using them.